pkgs/by-name/av/avbroot/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, gitUpdater
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "avbroot";
+  version = "3.4.0";
+
+  src = fetchFromGitHub {
+    owner = "chenxiaolong";
+    repo = "avbroot";
+    rev = "v${version}";
+    hash = "sha256-RoeanW4vCFm7TCr8T8akNnDI10gqXsMjqq39Y/7LL9o=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
You need to commit the lockfile otherwise we trigger ifd

Reply:
Aw, that's a shame. Are there any alternatives? I just remember recently reading that the cargo lockfiles in nixpkgs account for a not-insignificant part of it's _slowdown._

Reply:
Either way, I've fixed this. Please give it a review again if you have the time :)